docs/upgrade_to_tails_2x.rst

```diff
@@ -0,0 +1,138 @@
+Upgrade Tails from 1.x to 2.x
+=============================
+
+Newer versions of SecureDrop leverage improvements to the Tails OS since the introduction of Tails 2.0 (and subsequent versions). It is critical to upgrade all Tails USBs to the latest version before upgrading SecureDrop. For some admins, this might be complicated, since the upgrade from 1.x to 2.x must be done manually. Please follow this guide to updating the entire system, and special attention to only use the Secure Viewing Station computer so you benefit from its airgap while transfering sensitive data.
```

Comment:
I would also add a note saying that Tails 1.x is no longer receiving updates, so it is vulnerable to a growing list of security vulnerabilities. Everyone should upgrade to Tails 2.x, which will be getting automatic upgrades for the foreseeable future.